websites/N/NiceHash/presence.ts

```diff
@@ -0,0 +1,33 @@
+const presence = new Presence({
+    clientId: "915811466353983519"
+  }),
+    strings = presence.getStrings({
+      play: "presence.playback.playing",
+      pause: "presence.playback.paused"
+    }),
+  browsingTImestamp = Math.floor(Date.now() / 1000);
+
+  presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+    largeImageKey: "nicehash_1024x1024",
+    startTimestamp: browsingTImestamp
```

Comment:
that's not how you camelcase
```suggestion
    startTimestamp: browsingTimestamp
```